test/xcode-specs.js

```diff
@@ -89,7 +89,15 @@ describe('xcode @skip-linux', function () {
     (parseFloat(version)-6.1).should.be.at.least(0);
   });
 
-  it('should get a list of iOS devices', async () => {
+  it('should get max tvOS SDK version', async () => {
+    let version = await xcode.getMaxTVOSSDK();
+
+    should.exist(version);
+    (typeof version).should.equal('string');
+    (parseFloat(version)-6.1).should.be.at.least(0);
```

Comment:
It's checking if the minimum version is 6.1. It's related to your top comment regarding `version[0] === '4'`.